test/extended/deployments/deployments.go

```diff
@@ -20,7 +20,7 @@ import (
 	e2e "k8s.io/kubernetes/test/e2e/framework"
 
 	appsapi "github.com/openshift/origin/pkg/apps/apis/apps"
-	appsutil "github.com/openshift/origin/pkg/apps/util"
+	appsinternalutil "github.com/openshift/origin/pkg/apps/controller/util"
```

Comment:
in general extended tests should use external versions. seem like the functions needed are usually simple printf wrapper, maybe belonging to another package

Reply:
(or we should drop those wrappers entirely)

Reply:
+1